docs/local_install_docker_compose.md

```diff
@@ -1,102 +1,139 @@
 127.0.0.1       substra-backend.owkin.xyz
 ```
 
-## Get source code
+## 3. Get source code
 
-- Define a root directory for all your Substra git repositories (for instance `~/substra`)
+- Define a root directory for all your Substra git repositories, for instance `~/substra`:
 
-```bash
+```sh
 export SUBSTRA_SOURCE=~/substra
 mkdir -p $SUBSTRA_SOURCE
 cd $SUBSTRA_SOURCE
 ```
 
-- Clone the following repositories from github and fetch the dev branch:
+- Clone the following repositories from [Substra's Github](https://github.com/SubstraFoundation):
   - hlf-k8s
   - substra-chaincode
   - substra-backend
   - substra-frontend
 
-```bash
-# for instance, from github through https connection
-git clone -b dev https://github.com/SubstraFoundation/hlf-k8s.git
-git clone -b dev https://github.com/SubstraFoundation/substra-chaincode.git
-git clone -b dev https://github.com/SubstraFoundation/substra-backend.git
-git clone -b dev https://github.com/SubstraFoundation/substra-frontend.git
-```
-
-## Install command line interface
+```sh
+RepoToClone=(
+#!/bin/bash
+# https
+https://github.com/SubstraFoundation/hlf-k8s.git
+https://github.com/SubstraFoundation/substra-chaincode.git
+https://github.com/SubstraFoundation/substra-backend.git
+https://github.com/SubstraFoundation/substra-frontend.git
+
+# ssh
+# git@github.com:SubstraFoundation/hlf-k8s.git
+# git@github.com:SubstraFoundation/substra-chaincode.git
+# git@github.com:SubstraFoundation/substra-backend.git
+# git@github.com:SubstraFoundation/substra-frontend.git
+)
+for repo in ${RepoToClone[@]}
+do
+    echo "Cloning" $repo
+    git clone $repo
+done
+```
+TODO: ADD script?
+
+## 4. Install command line interface
 
 - It is recommended to create a python virtual environment to install the client dependencies.
 
 For instance with [virtualenv](https://virtualenv.pypa.io/en/latest/):
 
-```bash
+```sh
 python3 -m venv substra
 source substra/bin/activate
+# or source venv/bin/activate.fish if you are an awesome fish shell user :)
 ```
 
 - Install Substra python sdk and Substra command line interface.
 
-```
-pip install substra
+TODO: ADD requirements.txt! & `pip3 install -r requirements.txt`
+
+```sh
+pip3 install substra pyyaml termcolor
 ```
 
 - The Substra CLI should have been installed. To check the installation has been successful, launch the `substra --version` command. The following lines should be displayed:
 
-```bash
-$ substra --version
-0.1.0
+```sh
+substra --version
+0.3.0
 ```
 
-## Start the Substra network
+## 5. Start Substra
 
-- Install python scripts dependencies:
+### Start the Substra network
 
-```bash
-pip install pyyaml
-```
+- Build the network images and start the containers. To do so, launch the python `start.py` script:
 
-- Build the network images and start the containers. To do so, launch the python start.py script:
+> ???Note: for now, `start.py` will only work when your repositories are located at the root of the `home` of your machine (`~/substra`)
```

Comment:
Are you sure about this ? Isn't it that all other repo must be on the same level in your folder architecture ?

Reply:
My bad, thank you!